inc/saiswitch.h

```diff
@@ -412,6 +412,12 @@ typedef enum _sai_switch_attr_t
        Default no map */
     SAI_SWITCH_ATTR_QOS_TC_AND_COLOR_TO_DSCP_MAP,
 
+    /** Number of preamble and Inter frame gap bytes to be added in the 
+     *  Meters(Policer, shaper and meter counters) [uint32_t]. 
+     *  Default to 20 Bytes. 
+     */
+    SAI_SWITCH_ATTR_QOS_METER_ADJUST,
```

Comment:
can you explain a little bit more about this attribute? why 20 bytes, why this can be different on different platforms?